src/object.c

```diff
@@ -896,6 +896,39 @@ int getLongLongFromObjectOrReply(client *c, robj *o, long long *target, const ch
     return C_OK;
 }
 
+int getLongUnsignedIntFromObject(robj *o, uint64_t *target) {
```

Comment:
i'm not certain we actually need it.
the range of `long long` is high enough so that we can assume we'll never wrap around and get to negative numbers.
i'd rather drop these, but if we keep them they should probably use `unsigned long long` and not `uint64_t`

Reply:
Right, long long is enough. I will remove that